app/models/miq_policy.rb

```diff
@@ -355,6 +355,22 @@ def self.all_policy_events
     MiqEventDefinition.all_events.select { |e| !e.memberof.empty? && !EVENT_GROUPS_EXCLUDED.include?(e.memberof.first.name) }
   end
 
+  def self.all_policy_events_filter
+    # Todo Convert to SQL if possible
```

Comment:
@kbrock Can you take a look at this expression we came up with to get the list of seeded policy event definitions and see if it can be done in pure SQL? This new code creates a filter expression that can be used in a view, driven by reporting, get the same output [this method](https://github.com/ManageIQ/manageiq/blob/080e97baf85ad5a757ebb929ab38a8d4919812f6/app/models/miq_policy.rb#L354-L356) returns.

Reply:
@kbrock Can you look/comment on ^^ when you get a chance, thanks